### ISSUE TITLE
bullseye-arrow-missed

### ISSUE THREAD

**Issue by eviralpatel** [Templarian/MaterialDesign#5247]
I have:

- [YES] [Searched all issues](https://github.com/Templarian/MaterialDesign/issues) to make sure there isn't a request for this icon.
- [YES] [Searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't exist.
- [YES] Only requested a single icon in this issue. 

## Usage

bullseye-arrow-missed

## Examples

Currently there is an icon called bullseye-arrow. I am requesting an icon that can be called bullseye-arrow-missed. This icon should be very similar to the existing bullseye-arrow icon BUT the arrow should not point to the center, it should point to somewhere else to show that the aim is missed.

**Comment by Templarian** [comment]
![image](https://user-images.githubusercontent.com/338885/85648755-1fd28b00-b667-11ea-9693-6a41b6fab32a.png)

**Comment by Templarian** [comment]
@eviralpatel Not 100% on this one, wondering if it needs the dot also. Right now it might look like arrow+bullseye, rather than missing.

**Comment by eviralpatel** [comment]
Yes, you are right. this looks something else. It looks like arrow+bullseye but do not express any meaning to viewer.

Here is what I created using MS Paint - so its not perfect but just wanted to share my thoughts. [https://ibb.co/G5y0zcS](url)

1. The dot in the middle is OK. In fact we need to add one more circle. Currently there are two circles and a dot. Three circles and a dot will give more impression of a bullseye.
2. We need to make the arrow hit anywhere on the middle horizontal line of the circle.
3. If we can reduce the thickness of the arrow's transparent border then it will give viewer more idea of what is this icon about.

Above are just my two cents. I understand the constraints you might have.

Thanks a ton for all your efforts.
Viral Patel
